lib/chef/resource/apt_repository.rb

```diff
@@ -405,18 +465,20 @@ def make_ppa_url(ppa)
         # @param [Array] components
         # @param [Boolean] trusted
         # @param [String] arch
+        # @param [String] signed_by
         # @param [Array] options
         # @param [Boolean] add_src
         #
         # @return [String] complete repo config text
-        def build_repo(uri, distribution, components, trusted, arch, options, add_src = false)
+        def build_repo(uri, distribution, components, trusted, arch, signed_by, options, add_src = false)
```

Comment:
I get a bit of a twitchiness with the insert of a parameter in the "middle" of the `build_repo` interface, but I think it's "ok" as an internal API, and technically "options" is a parameter that would have a default value in other interfaces. Confirmed that `build_repo`'s only consumer outside of class itself is the `apt_repository_spec.rb`